cypress/integration/checkout/checkout.multipiece.items.spec.js

```diff
@@ -117,15 +117,15 @@ describe('Check Out', () => {
   });
 
   const fullCheckOut = ({ barcode, numberOfPieces, descriptionOfPieces, numberOfMissingPieces, missingPieces }) => {
-    CheckOutActions.checkOutItem(userBarcode, barcode);
+    CheckOutActions.checkOutItemUser(userBarcode, barcode);
```

Comment:
why we need to rename function?

Reply:
Bc there is already checkoutItem function, that checkouts only item. Is it ok for you?